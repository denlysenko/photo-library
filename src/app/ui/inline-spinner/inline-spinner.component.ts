import { ChangeDetectionStrategy, Component } from '@angular/core';

@Component({
  selector: 'app-inline-spinner',
  templateUrl: './inline-spinner.component.html',
  styleUrls: ['./inline-spinner.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class InlineSpinnerComponent {}
