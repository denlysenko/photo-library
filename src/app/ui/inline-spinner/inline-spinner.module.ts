import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';

import { InlineSpinnerComponent } from './inline-spinner.component';

@NgModule({
  imports: [CommonModule],
  declarations: [InlineSpinnerComponent],
  exports: [InlineSpinnerComponent],
})
export class InlineSpinnerModule {}
