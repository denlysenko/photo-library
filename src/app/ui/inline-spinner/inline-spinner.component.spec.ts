import { CommonModule } from '@angular/common';
import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { InlineSpinnerComponent } from './inline-spinner.component';

describe('InlineSpinnerComponent', () => {
  let component: InlineSpinnerComponent;
  let fixture: ComponentFixture<InlineSpinnerComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [CommonModule],
      declarations: [InlineSpinnerComponent],
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(InlineSpinnerComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture).toMatchSnapshot();
  });
});
