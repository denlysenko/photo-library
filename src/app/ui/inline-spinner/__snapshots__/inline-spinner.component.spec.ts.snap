// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineSpinnerComponent should create 1`] = `
<app-inline-spinner>
  <div
    class="ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</app-inline-spinner>
`;
